Userland/Libraries/LibCore/Process.h

```diff
@@ -8,20 +8,62 @@
 
 #pragma once
 
+#include <AK/DeprecatedString.h>
 #include <AK/Forward.h>
 #include <AK/Span.h>
+#include <LibCore/File.h>
 
 namespace Core {
 
+namespace FileAction {
+
+struct OpenFile {
+    DeprecatedString path;
+    File::OpenMode mode;
+    int fd;
```

Comment:
`fd` should be default init to -1. and open mode to ... something? Gotta explicitly default init all those primitive types.